### DISCUSSION TITLE
Oracle DB [Nov 05]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4378] [Discord Threads]
> 🙋‍♂️ **xp4each**

**Comment by o1lab** [comment]
> 🙋‍♂️ **wkw**  
#### we don't support oracle db at this moment. only sqlite, mysql, postgres and mssql.  
#  
> 💁 **o1lab**  
#### oracle is in the making & pls DM for more details  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1035592617594982420)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
